### ISSUE TITLE
The data is not loading since 2022.11.0 update

### ISSUE THREAD

**Issue by Efekent** [mrtnbr/ha_spaarnelanden_inzameling#4]
The integration was working perfectly till the new HA update 2022.11.0 (and 2022.11.1)

Integration tries to load constantly.

I am getting following error:
```
Deze fout is ontstaan door een aangepaste integratie.

Logger: homeassistant.helpers.entity
Source: custom_components/spaarnelanden_inzameling/sensor.py:166
Integration: spaarnelanden_inzameling (documentation)
First occurred: 16:44:46 (475 occurrences)
Last logged: 20:41:47

Update for sensor.spaarnelanden_inzameling_container_6078 fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 527, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 722, in async_device_update
    raise exc
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 192, in wrapper
    result = method(*args, **kwargs)
  File "/config/custom_components/spaarnelanden_inzameling/sensor.py", line 166, in update
    self._state = self.containerdata['filling_degree']
TypeError: 'bool' object is not subscriptable
```

Thanks in advance!

**Comment by mrtnbr** [comment]
There were changes on the source website. Fixed in 0.0.2 release

**Comment by Efekent** [comment]
@mrtnbr Thank you very much! It is working like a charm again. 

For those with this issue, once you update through HACS don't forget to put your container number in the sensor.py
